city_scrapers/spiders/chi_north_river_mental_health.py

```diff
@@ -0,0 +1,138 @@
+            for item in response.xpath('.//div[@class="wsite-section-elements"]//a'):
+
+                valid_start = self._minutes_parse_start(item)
+                if not valid_start:
+                    continue
+
+                meeting = Meeting(
+                    title="North River EMHSP governing commission",
+                    description="",
+                    classification=COMMISSION,
+                    start=valid_start,
+                    end=None,
+                    all_day=False,
+                    time_notes="",
+                    location=self._minutes_parse_location(item),
+                    links=self._minutes_parse_links(item, response),
+                    source=response.url,
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+        else:
+
+            item = response.css("td:nth-child(2) div:nth-child(2)")
+            meeting = Meeting(
+                title="North River EMHSP governing commission",
+                description="",
+                classification=COMMISSION,
+                start=self._index_parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self._index_parse_location(item),
+                links=self._index_parse_links(item, response),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _minutes_parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        date_components = item.re(
+            "(?P<month>[A-Za-z\\.]+)\\ "
+            "(?P<day>[0-9]+)(th)?,((\\ |\\xa0)?)"
+            "(?P<year>[0-9]+)?"
+        )
+
+        try:
+            if date_components[0] == "anuary":
+                date_components[0] = "January"
+
+            date_str = (
+                f"{date_components[-1]} {date_components[0]} {date_components[1]}"
+            )
+
+            if date_str == "20 May 20":
+                date_str = "20 May 2015"
+            elif date_str == " September 21":
+                date_str = "2016 September 21"
+            elif date_str == "201 July 15":
+                date_str = "2015 July 15"
+
+            return dateutil.parser.parse(f"{date_str} 7PM")
+        except IndexError:
+            return None
+
+    def _index_parse_start(self, item):
+        date_components = item.re(
+            "<br>Date: (?P<date>.*)<br>Time: (?P<time>.*)<br>Place:"
+        )
+        return dateutil.parser.parse(" ".join(date_components))
+
+    def _minutes_parse_location(self, item):
+        """Parse or generate location."""
+        start = self._minutes_parse_start(item)
+        if start < datetime.datetime(2017, 3, 15, 0, 0):
```

Comment:
Small thing, but you can leave out the zeros for hour and minute since they're defaults